index.html

```diff
@@ -279,8 +279,9 @@
 allow an end user to select zero ore more serial ports on the device - or to
 reject the request to access any ports.
 
-It is also RECOMMENDED that the user be shown all serial ports that are
-currently busy, either as a result of prior calls to `requestPorts()` or are in
+It is also RECOMMENDED that the user can select from all serial ports available
```

Comment:
Maybe change to, as conformance requirements apply to user agents (and it's better to be explicit): 

> It is also RECOMMENDED that the user agent allow the user to select from all serial ports available

Or, a bit shorter: 

> A user agent SHOULD allow the user to select from all serial ports available